content/applications/websites/ecommerce/maximizing_revenue/cross_selling.rst

```diff
@@ -1,55 +1,64 @@
-=============================================================
-How to sell accessories and optional products (cross-selling)
-=============================================================
+================================================
+Cross-selling: Accessories and Optional Products
+================================================
 
-You sell computers. Why not stimulating your customers to buy a top-notch
-screen or an extra-warranty? That's the goal of cross-selling functionalities: 
+When selling products, it is natural to look for new ways to stimulate customers to buy more goods,
```

Comment:
Any sales process is an opportunity to stimulate up-selling. For example, with an extra-warranty or some accessories.